tests/e2e/vsan_policy_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test suite tries to add, remove and replace vm to the _DEFAULT vmgroup
+// Expected behavior is that add/rm/replace vm for _DEFAULT vmgroup should fail
+
+package e2e
+
+import (
+	"log"
+	"os"
+
+	con "github.com/vmware/docker-volume-vsphere/tests/constants/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/admincli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/inputparams"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/verification"
+	. "gopkg.in/check.v1"
+)
+
+type VsanPolicyTestSuite struct {
+	esxIP  string
+	hostIP string
+}
+
+func (s *VsanPolicyTestSuite) SetUpSuite(c *C) {
+	s.hostIP = os.Getenv("VM1")
```

Comment:
henceforth, we plan to use the common test config introduced in #1286 . You can refer vmgroup tests to see how it that config is consumed.